keps/sig-release/2572-release-cadence/README.md

```diff
@@ -0,0 +1,389 @@
+| 1  | 1 | 1 (January 3) | |
+| 12 | 1 | 12 (March 15) | |
+| 14 | 2 | 1 (March 28) | Probable KubeCon EU |
+| 26 | 2 | 12 (June 28) | |
+| 28 | 3 | 1 (July 11) | | 
+| 40 | 3 | 12 (October 4) | | 
+| 42 | 4 | 1 (October 17) | Probably KubeCon NA | 
+| 52 | 4 | 10 (Dec 28) | | 
+
+This KEP proposes a transition to a *3-releases-per-year cadence*, beginning 
+with the Kubernetes 1.22 Release. This would result in a *15* week release 
+cycle, with *2* weeks between release cycles.
+
+*Kubernetes Release Schedule 2021 (Proposed 3 Release Cadence)*
+
+| Year Week Number | Release Number | Release Week | Note |
+| -------- | -------- | -------- | -------- |
+| 2  | 1 | 1 (January 11) | |
+| 14 | 1 | 13 (April 8) | |
+| 17 | 2 | 1 (April 26) | |
+| 32 | 2 | 15 (August 10) | KubeCon EU - 14 weeks of actual work|
+| 35 | 3 | 1 (August 31) | | 
+| 50 | 3 | 15 (December 14) | Kubecon NA - 14 weeks of actual work | 
+
+*Kubernetes Release Sechedule 2022 (Proposed 3 Release Cadence)*
+
+| Year Week Number | Release Number | Release Week | Note |
+| -------- | -------- | -------- | -------- |
+| 1  | 1 | 1 (January 3) | |
+| 15 | 1 | 15 (April 12) | | 
+| 18 | 2 | 1 (May 2) | Probably KubeCon EU |
+| 33 | 2 | 15 (August 15) | |
+| 36 | 3 | 1 (September 6 | Probably KubeCon NA | 
+| 51 | 3 | 15 (December 20) | 
+
+### User Stories
+
+Kubernetes releases are made by real people. The technical aspects—for example,
+the release automation—reflects only a tiny part of the complete cycle. This
+means we will mainly focus on the human aspects and their corresponding roles
+when deciding to move to a 3-releases-per-year cadence.
+
+#### End User
+
+Most end user organizations find it difficult to match Kubernetes release
+cadence - only 3 releases per year will relax this situation.
+
+#### Distributors and downstream projects
+
+Downstream projects assemble their solution from different projects. Having
+fewer upgrades helps them to reduce complexity. For example, cloud providers
+will gain more room for upgrading their infrastructure.
+
+#### Contributors
+
+With a lower release cadence, contributors will gain more time for project
+enhancements, feature development, planning, and testing. It will provide more
+room for maintaining their mental health, prepare for events like KubeCon or
+work on the downstream integration.
+
+Through this proposal SIG Release's aim is to give contributors more
+flexibility to decide how to invest their time. It is explicitly _not_ to push
+contributors in doing more.
+
+#### SIG Release members
+
+By applying a cadence of 3 releases per year, SIG Release members will gain a
+reduced management overhead. There are also only 3 patch releases to maintain,
+which right now can overlap up to 4. SIG Release will gain more time to ensure
+a seamless transition from the previous release team to the next one. It is
+also possible to include more shadows if the role leads conclude that this is
+appropriate.
+
+### Risks and Mitigations
+
```

Comment:
I'd like to see a Risk addressing not adequately communicating the decision in advance and what steps we're taking to ensure this isn't disruptive to the community & downstream.

Reply:
```suggestion
#### Late Notification of Cadence Change to Community and Downstream

This KEP is proposing changing the default release cadence from 3 to 4 during the period of time between releases. There is a risk that changing the default cadence will minimal notice will be disruptive to both the community and downstream. The notification and adjustment period will be very short (<1 week) for those expecting a traditional 4 cadence a year release schedule which would be expected to begin the week of April 19th.
```

Reply:
@jeremyrickard PTAL I added language as requested, but unsure what the mitigation strategy for this is.

Reply:
@kikisdeliveryservice I think we're all in agreement that it's too late to start with 1.22. Do you think the same content would be applicable if we apply the change in 1.23?

Alpha (1.22): communication, mitigate the downstream risks as much as possible through:
* blog post 
* community meeting (if they occur)
* k8s contributor tweet

Beta (1.23): make the transition to 3 release cadence (effectively just not doing the shortened 1.24)

Stable (which ever release that should be, if the requirement that things are "beta" for two releases applies, then 1.25).

Reply:
> I think we're all in agreement that it's too late to start with 1.22.

Too late to start the four month cadence? We started that with 1.19.

Reply:
@liggitt I meant "officially" without proper communication and agreement from folks. The reality is exactly what you just stated.

Reply:
@jeremyrickard I think the alpha comms would be a (sort of) mitigation :+1:

Reply:
looks like a pending discussion to be resolved here?

Reply:
@kikisdeliveryservice you ok to resolve this one now? we added some concrete communication avenues.